packages/orchestrator-ui-components/src/components/WfoForms/formFields/SubscriptionField.tsx

```diff
@@ -0,0 +1,380 @@
+                tags &&
+                tags?.length > 0 &&
+                !tags.includes(subscription.product.tag)
+            ) {
+                return false;
+            }
+
+            // If specific subscriptionIds are excluded the subscription can't be one ot those
+            if (
+                excludedSubscriptionIds &&
+                excludedSubscriptionIds.length > 0 &&
+                excludedSubscriptionIds.includes(subscription.subscriptionId)
+            ) {
+                return false;
+            }
+
+            // If a Port mode filter is applied we need to filter on that
+            if (visiblePortMode !== 'all') {
+                const portMode = getPortMode(
+                    subscription.productBlockInstances,
+                );
+                // For normal mode filter out all subscriptions that don't have tagged or untagged ports
+                if (
+                    visiblePortMode === 'normal' &&
+                    ![PortMode.TAGGED, PortMode.UNTAGGED, undefined].includes(
+                        portMode,
+                    )
+                ) {
+                    return false;
+                } else if (portMode !== visiblePortMode) {
+                    return false;
+                }
+            }
+
+            // If a customer/organisation filter is applied we need to filter on that
+            if (
+                usedOrganisationId &&
+                subscription.customer?.identifier !== usedOrganisationId
+            ) {
+                return false;
+            }
+
+            if (parentName !== name) {
+                if (
+                    parent.fieldType === Array &&
+                    // @ts-ignore
+                    parent.uniqueItems
+                ) {
+                    const allValues: string[] = get(model, parentName, []);
+                    const chosenValues = allValues.filter(
+                        (_item, index) =>
+                            index.toString() !==
+                            nameArray[nameArray.length - 1],
+                    );
+                    if (!chosenValues.includes(subscription.subscriptionId)) {
+                        return false;
+                    }
+                }
+            }
+
+            return true;
+        });
+
+        return filteredSubscriptions
+            ? filteredSubscriptions.map((subscription) => ({
+                  label: makeLabel(subscription),
+                  value: subscription.subscriptionId,
+              }))
+            : [];
+    };
+
+    const options = getSubscriptionOptions();
+    const selectedValue = options.find(
+        (option: Option) => option.value === value,
+    );
+
+    const isDisabled = disabled || readOnly || isFetching;
+
+    return (
+        <EuiFlexItem css={subscriptionFieldStyling} grow={1}>
+            <section
+                {...filterDOMProps(props)}
+                className={`${className} subscription-field${
```

Comment:
You can see how I fixed the styling of acceptField. Probably the same pattern can be applied here.